experiment_configs/lock_in_keithley_gating_oussama.json

```diff
@@ -0,0 +1,13 @@
+{
+    "gate_start": 0,
+    "gate_end": 2,
+    "gate_steps": 10,
+    "voltage": 0.004, 
+    "frequency": 10000, 
+    "sensitivity": 20, 
+    "time_constant": 0, 
+    "slope": 1,
+    "ch_id_1": "R", 
```

Comment:
If these are already strings then there is no need for `_SRS_lock_in_data_key` above.